src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -220,34 +240,32 @@ public Tensor ProcessImage(string path)
                     labelColumnName, typeof(uint).ToString(),
                     labelColumn.Type.RawType.ToString());
 
-            var imagePathColumn = input.Schema[imagepathColumnName];
+            var imageBufColumn = input.Schema[imageColumnName];
```

Comment:
>imageBufColumn [](start = 16, length = 14)

Please stop referring "Buffer" as "Buf". In this case just call it as "ImageColumn" and be done with it.

Reply:
Addressed in my latest commit. Thanks.